DynamicSize/DynamicSizeSnapshot.swift

```diff
@@ -164,13 +166,14 @@ func performDynamicSizeSnapshotTest(_ name: String?, sizes: [String: CGSize], is
     let result = sizes.map { (sizeName, size) -> Bool in
         // swiftlint:disable:next force_unwrapping
         let view = instance.snapshotObject!
+        let _snapshotName = identifier != nil ? "\(snapshotName)_\(identifier!) - \(sizeName)" : "\(snapshotName) - \(sizeName)"
```

Comment:
Let's replace this force-unwrap with something like:

```swift
let _snapshotName

if let identifier = identifier {
    _snapshotName = "\(snapshotName)_\(identifier) - \(sizeName)"
} else {
    _snapshotName = "\(snapshotName) - \(sizeName)"
}
```

Reply:
Done!! It was in my refactor and I forgot it. 👍